ERROR - 2022-02-04 11:34:40 --> ErrorException: pg_query(): Query failed: ERROR:  database "food" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(221): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\CreateDatabase.php(137): CodeIgniter\Database\Forge->createDatabase()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\CreateDatabase->run()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2022-02-04 11:39:29 --> ErrorException: pg_query(): Query failed: ERROR:  database "project_food" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(221): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\CreateDatabase.php(137): CodeIgniter\Database\Forge->createDatabase()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\CreateDatabase->run()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#14 {main}
ERROR - 2022-02-04 12:26:54 --> ErrorException: pg_query(): Query failed: ERROR:  relation "usuarios" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(513): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Migrations\2022-02-04-174227_CriaTabelaUsuarios.php(85): CodeIgniter\Database\Forge->createTable()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(863): App\Database\Migrations\CriaTabelaUsuarios->up()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2022-02-04 12:45:40 --> ErrorException: pg_query(): Query failed: ERROR:  relation "usuarios" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(513): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Migrations\2022-02-04-174227_CriaTabelaUsuarios.php(87): CodeIgniter\Database\Forge->createTable()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(863): App\Database\Migrations\CriaTabelaUsuarios->up()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2022-02-04 12:50:41 --> ErrorException: pg_query(): Query failed: ERROR:  null value in column "password" of relation "usuarios" violates not-null constraint
DETAIL:  Failing row contains (1, Alberto Cesar Marques Amorim, Albertocesar@hotmail.com, 495.513.710-54, 81 - 99609-8382, f, f, null, null, null, null, null, null, null). in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Seeds\UsuarioSeeder.php(21): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-02-04 12:54:00 --> ErrorException: pg_query(): Query failed: ERROR:  column "cpf" of relation "usuarios" does not exist
LINE 1: INSERT INTO "usuarios" ("nome", "email", "cpf", "telefone") ...
                                                 ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Seeds\UsuarioSeeder.php(21): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-02-04 12:54:03 --> ErrorException: pg_query(): Query failed: ERROR:  relation "usuarios" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(513): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Migrations\2022-02-04-174227_CriaTabelaUsuarios.php(87): CodeIgniter\Database\Forge->createTable()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(863): App\Database\Migrations\CriaTabelaUsuarios->up()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2022-02-04 13:05:32 --> ErrorException: pg_query(): Query failed: ERROR:  column "cpf" of relation "usuarios" does not exist
LINE 1: INSERT INTO "usuarios" ("nome", "email", "cpf", "telefone") ...
                                                 ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Seeds\UsuarioSeeder.php(21): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-02-04 13:11:19 --> ErrorException: pg_query(): Query failed: ERROR:  column "cpf" of relation "usuarios" does not exist
LINE 1: INSERT INTO "usuarios" ("nome", "email", "cpf", "telefone") ...
                                                 ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1771): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(239): CodeIgniter\Database\BaseBuilder->insert()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(738): CodeIgniter\Model->doInsert()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(625): CodeIgniter\BaseModel->insert()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Seeds\UsuarioSeeder.php(21): CodeIgniter\Model->insert()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Seeder.php(147): App\Database\Seeds\UsuarioSeeder->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Seed.php(77): CodeIgniter\Database\Seeder->call()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Seed->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#17 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#18 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#19 {main}
ERROR - 2022-02-04 13:11:29 --> ErrorException: pg_query(): Query failed: ERROR:  relation "usuarios" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(513): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Migrations\2022-02-04-174227_CriaTabelaUsuarios.php(87): CodeIgniter\Database\Forge->createTable()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(863): App\Database\Migrations\CriaTabelaUsuarios->up()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2022-02-04 13:19:57 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:19:57 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:20:06 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:20:06 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:21:09 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:21:09 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:22:04 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:22:04 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:23:35 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:23:35 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:23:58 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:23:58 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(19): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:37:41 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:37:41 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:43:47 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:43:47 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
ERROR - 2022-02-04 13:45:21 --> ErrorException: pg_query(): Query failed: ERROR:  relation "usuarios" already exists in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Forge.php(513): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Database\Migrations\2022-02-04-174227_CriaTabelaUsuarios.php(87): CodeIgniter\Database\Forge->createTable()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(863): App\Database\Migrations\CriaTabelaUsuarios->up()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\MigrationRunner.php(186): CodeIgniter\Database\MigrationRunner->migrate()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Database\Migrate.php(84): CodeIgniter\Database\MigrationRunner->latest()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Commands.php(63): CodeIgniter\Commands\Database\Migrate->run()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(70): CodeIgniter\CLI\Commands->run()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\CommandRunner.php(56): CodeIgniter\CLI\CommandRunner->index()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(823): CodeIgniter\CLI\CommandRunner->_remap()
#13 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#14 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#15 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CLI\Console.php(48): CodeIgniter\CodeIgniter->run()
#16 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\spark(63): CodeIgniter\CLI\Console->run()
#17 {main}
ERROR - 2022-02-04 13:48:30 --> ErrorException: pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^ in C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php:135
Stack trace:
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:48:30 --> pg_query(): Query failed: ERROR:  column usuarios.deleted_at does not exist
LINE 3: WHERE &quot;usuarios&quot;.&quot;deleted_at&quot; IS NULL
              ^
#0 [internal function]: CodeIgniter\Debug\Exceptions->errorHandler()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\Postgre\Connection.php(135): pg_query()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(670): CodeIgniter\Database\Postgre\Connection->execute()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseConnection.php(598): CodeIgniter\Database\BaseConnection->simpleQuery()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Model.php(183): CodeIgniter\Database\BaseBuilder->get()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\BaseModel.php(584): CodeIgniter\Model->doFindAll()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(21): CodeIgniter\BaseModel->findAll()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#9 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#10 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#11 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#12 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#13 {main}
CRITICAL - 2022-02-04 13:49:01 --> Invalid file: Admin/Usuarios/index.php
#0 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Common.php(1089): CodeIgniter\View\View->render()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(24): view()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#8 {main}
CRITICAL - 2022-02-04 13:51:58 --> Invalid file: Admin/home/layout/principal.php
#0 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile()
#1 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\View\View.php(226): CodeIgniter\View\View->render()
#2 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Common.php(1089): CodeIgniter\View\View->render()
#3 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\app\Controllers\Admin\Usuarios.php(24): view()
#4 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(825): App\Controllers\Admin\Usuarios->index()
#5 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController()
#6 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest()
#7 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\public\index.php(37): CodeIgniter\CodeIgniter->run()
#8 C:\Users\Adasi\Documents\project_food\Aula_codeigniter\system\Commands\Server\rewrite.php(43): require_once('C:\\Users\\Adasi\\...')
#9 {main}
